test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -424,7 +424,7 @@ void test_ucp_mmap::test_rkey_proto(ucp_mem_h memh)
     ASSERT_UCS_OK(status);
 
     /* Check rkey configuration */
-    if (!disable_proto() && m_ucp_config->ctx.proto_enable) {
+    if (!disable_proto() && is_proto_enabled()) {
```

Comment:
Guess that should be enough:
```suggestion
    if (is_proto_enabled()) {
```